signer/src/emily_client.rs

```diff
@@ -1,12 +1,24 @@
 //! Emily API client module
```

Comment:
Can you add a `TODO` to use the autogenerated `Emily` openapi client?

Reply:
Uhm we are using that (eg, the `deposit_api::get_deposits` below). Are you referring to something specific?